parser/parser_system1.c

```diff
@@ -1262,6 +1270,8 @@ static int parse_platform(struct pv_state *s, char *buf, int n)
 			      do_action_for_storage, false),
 		ADD_JKA_ENTRY("drivers", JSMN_OBJECT, &bundle,
 			      do_action_for_drivers, false),
+		ADD_JKA_ENTRY("exports", JSMN_ARRAY, &bundle,
```

Comment:
Is what we expect an array and not a string?

Reply:
I guess... as you notice we lack of docs about it, my only sources of docs are 

- https://github.com/pantavisor/pantavisor/blob/master/scripts/hooks_lxc-mount.d/export.sh
- https://gitlab.com/pantacor/pvr/-/blob/master/templates/builtin-lxc-docker.go?ref_type=heads